examples/all-clusters-app/nrfconnect/main/AppTask.cpp

```diff
@@ -59,10 +59,10 @@ K_MSGQ_DEFINE(sAppEventQueue, sizeof(AppEvent), APP_EVENT_QUEUE_SIZE, alignof(Ap
 
 namespace {
 
-// NOTE! This key is for test/certification only and should not be available in production devices.
-// Ideally, it should be a part of the factory data set.
-constexpr uint8_t kTestEventTriggerEnableKey[16] = { 0x00, 0x11, 0x22, 0x33, 0x44, 0x55, 0x66, 0x77,
-                                                     0x88, 0x99, 0xaa, 0xbb, 0xcc, 0xdd, 0xee, 0xff };
+// NOTE! This key is for test/certification only and should not be available in production devices!
+// To use production-ready Enable Key add it to the factory data and enable CONFIG_CHIP_FACTORY_DATA config.
```

Comment:
"production-ready" can be confusing given that we've just said that it's not supposed to be used in production. Maybe: "if `CONFIG_CHIP_FACTORY_DATA` is enabled, this value is read from the factory data". Also, nit: `static` is redundant if this is in anonymous namespace.